schema/fair.js

```diff
@@ -38,6 +40,20 @@ const FairType = new GraphQLObjectType({
         return `/${id}`;
       },
     },
+    image: Image,
+    location: {
```

Comment:
This is a weird one, when you fetch via `api/v1/fairs`, the short JSON doesn't include a location field. However, when fetching via `/api/v1/:fair/id`, the public JSON does include the location.

So we just resolve the location if it's present, otherwise we explicitly fetch the fair and resolve that location.

I verified that when requesting a specific fair, it only fetches it once, and when requesting the location of multiple fairs, it does indeed fetch each of them.